### DISCUSSION TITLE
Failed billing for copilot

### DISCUSSION THREAD

**Discussion by TNick** [community/community#69585] [New to GitHub] [answered]
![image](https://github.com/community/community/assets/1712330/596690e9-e7e3-4192-95c5-1c0df8d0ca44)
I've got an email that GitHub `had a problem billing your account ` (the card is active, has funds) and when I go to update the payment method I see this.

Anybody else experiencing this issue?

**Comment by gazwannagm** [comment, marked as answer]
For any account-specific queries like this, I'd recommend reaching out directly through [Support page](https://support.github.com/contact) by creating a support ticket. The team there should be able to assist and provide the guidance you need.

**Reply by Alvin798** [reply]
Stop all error messages

**Comment by ansovannath** [comment]
<img width="1203" height="652" alt="image" src="https://github.com/user-attachments/assets/cc110cb8-aa9e-45b2-b20a-63cfae1a614d" />
Why did the payment attempt fail on July 8, 2025? In the payment history, it always bills on the 13th or 14th. If I can't pay on time, this payment due will be charged again on the 21st . Why due payment doesn't sent to me

**Comment by operaplussyr-dotcom** [comment]
We are having a problem billing your account. Please [update your payment method](https://github.com/settings/billing/payment_information) or call your payment provider for details on why the transaction failed.
